Documentation/README.md

```diff
@@ -55,7 +55,7 @@ swift build
 
 Swift will build a single executable called `foo`.
 
-To the package manager, everything is a package, hence `Package.swift`. However, this does not mean you have to release your software to the wider world: you can develop your app without ever publishing it in a place where others can see or use. On the other hand, if one day you decide that your project _should_ be  available to a wider audience your sources are already in a form ready to be published.  The package manager is also independent of specific forms of distribution, so you can use it to share code within your personal projects, within your workgroup, team or company, or with the world.
+To the package manager, everything is a package, hence `Package.swift`. However, this does not mean you have to release your software to the wider world; you can develop your app without ever publishing it in a place where others can see or use it. On the other hand, if one day you decide that your project _should_ be  available to a wider audience your sources are already in a form ready to be published.  The package manager is also independent of specific forms of distribution, so you can use it to share code within your personal projects, within your workgroup, team or company, or with the world.
```

Comment:
Not sure if this is just me, but I think there a few extra spaces in this paragraph, for example before `The package manager is also..`.

Reply:
There were several of these throughout, as well as a number of hard tabs, and numerous trailing spaces. I installed a markup linter and caught, I believe, all of the whitespace related issues.